client/output.rs

```diff
@@ -185,7 +185,8 @@ pub fn print_logs(
 ) {
     let (json, task_ids) = match cli_command {
         SubCommand::Log { json, task_ids } => (*json, task_ids.clone()),
-        SubCommand::Follow { task_id, .. } => (false, vec![*task_id]),
+        // TODO: Check that SubCommand::Follow is also unreachable.
```

Comment:
What exactly do you mean by that?

Reply:
Well, from what I could tell, this function is only ever called when the logs are being printed out and thus the `Follow` sub-command should never appear.  I left it commented just in case I misunderstood the logic though.

Reply:
Nope, thats correct :)
This path should only be entered with `Log`